server/service/IndividualServicesService.js

```diff
@@ -360,27 +409,36 @@ exports.redirectInfoAboutApprovalStatusChanges = function (body, user, originato
       let applicationAddress = body["subscriber-address"];
       let applicationPort = body["subscriber-port"];
       let subscriberOperation = body["subscriber-operation"];
+      let applicationProtocol = body["subscriber-protocol"];
 
       /****************************************************************************************
        * Prepare logicalTerminatinPointConfigurationInput object to 
        * configure logical-termination-point
        ****************************************************************************************/
 
-      let operationList = [
-        subscriberOperation
+      let operationNamesByAttributes = new Map();
+      operationNamesByAttributes.set("subscriber-operation", subscriberOperation);
+
```

Comment:
These changes are not part of this pull request . Please remove them

Reply:
done